onnx/defs/tensor/defs.cc

```diff
@@ -260,6 +260,10 @@ ONNX_OPERATOR_SET_SCHEMA(
             return;
           }
 
+          if (targetShapeProto.dim_size() == 0) {
```

Comment:
Why can't we allow an empty shape? That's what we need for a scalar. So, if we want to convert a tensor with shape [1] into a scalar with shape [], I think that should be permitted?

Reply:
Thank you for catching it! I wasn't sure for the case of empty shape because the spec is not very clear. But, it makes sense to me that using empty shape for reshaping 1D to scalar. Will correct it and focus on the fix in version_converter.

Reply:
Thanks for clarifying the documentation. May be it will help to also say that "The input tensor's shape and the output tensor's shape are required to have the same number of elements."